scripts/temp-create-data.js

```diff
@@ -105,13 +107,15 @@ const createUser = async (username, accountIndex = 0) => {
   const userAddress = utils.getAddress(userWallet.address);
   userWallet.address = userAddress;
 
+  const thumbnail = username === 'a' ? thumbnailUserA : username === 'b' ? thumbnailUserB : thumbnailUserC;
+
   const userQuery = await graphqlRequest(
     createUniqueUser,
     {
       input: {
         id: userAddress,
         name: username,
-        profile: { displayName: `User ${username.toUpperCase()}` },
+        profile: { displayName: `User ${username.toUpperCase()}`, thumbnail: thumbnail },
```

Comment:
You import everything as a single object:
```ts
const userThumbnails = require('./imageData');
```

Then you assign them like this
```suggestion
        profile: { displayName: `User ${username.toUpperCase()}`, thumbnail: userThumbnails[`thumbnailUser${username.toUpperCase()}`] },
```

That way, you won't need the `const` assignment on line 110

Reply:
Great suggestion, thanks